Soomla/Assets/Plugins/Soomla/Profile/SoomlaProfile.cs

```diff
@@ -68,18 +71,27 @@ public class SoomlaProfile
 
 #if SOOMLA_FACEBOOK
 			unreadyProviders++;
-			providers.Add(Provider.FACEBOOK, new FBSocialProvider());
 #endif
 #if SOOMLA_GOOGLE
 			unreadyProviders++;
-			providers.Add(Provider.GOOGLE, new GPSocialProvider());
 #endif
 #if SOOMLA_TWITTER
 			unreadyProviders++;
-			providers.Add(Provider.TWITTER, new TwitterSocialProvider());
 #endif
 #if SOOMLA_GAMECENTER
 			unreadyProviders++;
+#endif
+
+#if SOOMLA_FACEBOOK
+			providers.Add(Provider.FACEBOOK, new FBSocialProvider());
+#endif
+#if SOOMLA_GOOGLE
+			providers.Add(Provider.GOOGLE, new GPSocialProvider());
+#endif
+#if SOOMLA_TWITTER
+			providers.Add(Provider.TWITTER, new TwitterSocialProvider());
+#endif
+#if SOOMLA_GAMECENTER
 			providers.Add(Provider.GAME_CENTER, new GameCenterSocialProvider());
 #endif
```

Comment:
@EugeneButusov please add a comment above this code to explain why you separated `unreadyProviders++;` from `providers.Add`